Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -848,6 +845,13 @@ spec:
       osd_max_scrubs: "10"
 ```
 
+If both these `cephConfig` and [ceph.conf settings](../../Storage-Configuration/Advanced/ceph-configuration/#custom-cephconf-settings)
+are applied, the `cephConfig` settings will take higher precedence if there is an overlap.
+
+If Ceph settings need to be applied to mons before quorum is initially created, the
+[ceph.conf settings](../../Storage-Configuration/Advanced/ceph-configuration/#custom-cephconf-settings)
+should be used instead.
+
```

Comment:
Perhaps we should say why `cephConfig` is preferred. E.g. "The operator will actively apply these values whereas the `rook-config-override` cm only has effect when pods are restarted?"  My opinion is that we should be encouraging the use of the new field over the cm.

Would it be possible to have the operator write out the `cm` from the `cephCluster.cephConfig` and relieve end users from directly managing it?

Reply:
> Perhaps we should say why `cephConfig` is preferred. E.g. "The operator will actively apply these values whereas the `rook-config-override` cm only has effect when pods are restarted?" My opinion is that we should be encouraging the use of the new field over the cm.

Agreed, I'll add that note as an important dinstinction.

> Would it be possible to have the operator write out the `cm` from the `cephCluster.cephConfig` and relieve end users from directly managing it?

I don't think we need to or want to overwrite the configmap. There is still the scenario of mons picking up these ceph.conf overrides before they are in quorum. Also, for the users still using the ceph.conf overrides, we wouldn't want to risk overriding their settings.